index.js

```diff
@@ -222,8 +228,13 @@ async function main() {
     .option('-f, --force', 'Force push to remote')
     .option('--push-merged', 'Push all branches (including those that have already been merged into the base branch)')
     .option('--remote <remote>', 'Set remote to push to. Defaults to "origin"')
-    .option('-b, --base <base>', `Specify the base branch to use for the first and combined PRs.`, defaultBase)
-    .option('-c, --create-prs', 'Create GitHub PRs from your train branches');
+    .option('-b, --base <base>', `Specify the base branch to use for the first and combined PRs.`, defaultBase);
+  if (draftByDefault) {
```

Comment:
Do you think it's necessary to have this `if` here? I think it might be better to simply always show all the options (so that the user knows what they can do) - I'd say it's somewhat unusual to change what flags are allowed based on what defaults are set?